contentcuration/contentcuration/models.py

```diff
@@ -1412,6 +1412,33 @@ def get_details(self, channel_id=None):
         else:
             channel = self.get_channel()
 
+        if not descendants:
```

Comment:
I don't think we reuse this query in this form, so probably don't need to do the full query here? Could we just do `descendants.exists()` instead?

Reply:
As usual, you are right, doing this way improves the performance. Done, thanks!